WikipediaUnitTests/Code/LocationManagerTests.swift

```diff
@@ -0,0 +1,144 @@
+import XCTest
+@testable import WMF
+
+final class LocationManagerTests: XCTestCase {
+
+    private var mock: MockCLLocationManager!
+    private var locationManager: WMFLocationManager!
+    private var delegate: LocationManagerTestsDelegate!
+
+    override func setUp() {
+        super.setUp()
+
+        MockCLLocationManager.resetMockData()
+        mock = MockCLLocationManager()
```

Comment:
`mock` is not enough. Maybe `mockCLLocationManager` is better. There are too many location managers and it's not clear what is what.